src/Microsoft.TryDotNet.WasmRunner/CodeRunner.cs

```diff
@@ -77,6 +77,8 @@ private static  Assembly LoadFromBase64EncodedString(string base64EncodedAssembl
             }
             catch (Exception e)
             {
+                onError(e.ToString());
+                onOutput(e.ToString());
```

Comment:
I'm not super familiar with this code - it is weird though that we have to call both onError and onOutput for an exception?

Reply:
I agree with @AbhitejJohn point. but the exception is wired on the run result, i think we are doing the wrong thing after the runner has produced the reunResult, probably in the integration part

Reply:
Just to clarify, the changes on this pr is only using onOutput(e.ToString());  
onError() was deleted.

@colombod do you still think the same ?

Reply:
@ocallesp : This still seem like the wrong place to make the fix since we aren't really passing the exception details through but just printing it to the output pane as Diego mentions. Can you please take a look at the suggestion above and re-work this? At the moment, this seems like a hack.